go-controller/pkg/ovn/gress_policy.go

```diff
@@ -123,8 +125,8 @@ func (gp *gressPolicy) addIPBlock(ipblockJSON *knet.IPBlock) {
 	gp.ipBlockExcept = append(gp.ipBlockExcept, ipblockJSON.Except...)
 }
 
-func ipMatch() string {
-	if config.IPv6Mode {
+func ipMatch(ip net.IP) string {
```

Comment:
This is now only used in one place and could just be inlined there

Reply:
done